.github/actions/atorch-python-test/action.yml

```diff
@@ -7,6 +7,8 @@ runs:
   args:
     - "/bin/bash"
     - "-c"
-    - "pip install pytest \
+    - "pip install pytest fairscale==0.4.1\
```

Comment:
fairscale==0.4.1 already included in requirments.txt, rebuild docker image will include it.